src/components/Form.jsx

```diff
@@ -1,43 +1,46 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
-import { addBook } from '../redux/books/books';
+import { addAPI } from '../redux/books/books';
 
 const Form = () => {
-  const [bookTitle, setTitle] = useState('');
-  const [bookAuthor, setAuthor] = useState('');
-
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
   const dispatch = useDispatch();
 
-  const submitForm = (event) => {
-    event.preventDefault();
-    dispatch(addBook({
-      title: bookTitle,
-      author: bookAuthor,
-    }));
+  const submitBookToStore = (e) => {
+    e.preventDefault();
+    const newBook = {
+      item_id: Date.now(),
+      title,
+      author,
+    };
+    dispatch(addAPI(newBook));
+    setTitle('');
+    setAuthor('');
   };
```

Comment:
Issues resolved. The add functionality is working. It can post to an external API now. I checked with Postman by sending a get request with this - 
https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/fu2GxwNDMRGqiSNEy6eo

![image](https://user-images.githubusercontent.com/30408871/185498951-1c53b089-0dc8-4760-b084-384153d55826.png)